### PR TITLE
Update accessing-a-sql-database-entities.md

### PR DESCRIPTION
**Pull request by GoFightNguyen**
# Fixed incorrect method indentation
## Summary

Correcting the indentation of the addInstructor function, which is the first code sample under the Updating the Database section.

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C3%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610031645391827-1106%5Cworkflow_report.html).

File: docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md
Comment (svick) [inline docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md]: Nit: I think these two lines should be indented too, so that they stay aligned with the first argument.

Comment (jasonsnguyen) [inline docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md]: I agree with you @svick. I made the change in my fork. Please advise on what I need to do next. This is my first attempt at a pull request.

Comment (svick) [inline docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md]: Now you just wait until someone from Microsoft (read: not me) comes along and either accepts the PR or tells you what else to improve in the PR (but with such a small PR, I think there's not much room for that).

**Comment by mairaw** [comment]
The changes look good. I'll wait for the CI builds to finish before merging this. Thanks @jasonsnguyen!

**Review by cartermp — APPROVED** [review]
Looks good to me.  If it looks fine on staging (I imagine it will), I'll merge it.  Thanks, @jasonsnguyen!

**Comment by cartermp** [comment]
Heh, @mairaw got to it first

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C3%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610031959160433-1106%5Cworkflow_report.html).

**Pull request merged**
